### PR TITLE
#2482 Add plugins tab in workspace and inside overview tab

### PR DESCRIPTION
**Pull request by wu87988622**
### Checklist
- [x] [Review Contribution Guidelines](https://github.com/oharastream/ohara/blob/master/CONTRIBUTING.md)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#authors))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #2482 )
- [ ] Add unit tests (or please explain why this PR is not worth having new tests)
- [x] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

**Comment by konekoya** [comment]
retry manager-e2e

File: ohara-manager/client/cypress/e2e/workspacesPage.test.js [hunk 1]
```diff
@@ -332,9 +340,9 @@ describe('WorkspacesPage', () => {
       .wait('@getFiles')
       .then(xhr => {
         cy.log(xhr);
```
Comment (konekoya) [inline ohara-manager/client/cypress/e2e/workspacesPage.test.js]: Let's remove the console in our production code

File: ohara-manager/client/src/components/pages/WorkspacesPage/WorkspacesDetailPage/Overview/OverviewTopics.js [hunk 2]
```diff
@@ -31,14 +31,12 @@ import * as useApi from 'components/controller';
...
+  const { handleRedirect, worker } = props;
   const { data: topics, isLoading: fetchingTopics } = useApi.useFetchApi(
-    URL.TOPIC_URL,
+    `${URL.TOPIC_URL}?group=${worker.name}-topic`,
   );
 
-  const topicsUnderBrokerCluster = get(topics, 'data.result', []).filter(
-    topic => topic.brokerClusterName === brokerClusterName,
-  );
+  const topicsUnderBrokerCluster = get(topics, 'data.result', []);
```
Comment (konekoya) [inline ohara-manager/client/src/components/pages/WorkspacesPage/WorkspacesDetailPage/Overview/OverviewTopics.js]: You can rename this to just `topics` now

File: ohara-manager/client/cypress/e2e/workspacesPage.test.js (hunk 1 above)
Comment (wu87988622) [reply ohara-manager/client/cypress/e2e/workspacesPage.test.js]: ok thanks

File: ohara-manager/client/src/components/pages/WorkspacesPage/WorkspacesDetailPage/Overview/OverviewTopics.js (hunk 2 above)
Comment (wu87988622) [reply ohara-manager/client/src/components/pages/WorkspacesPage/WorkspacesDetailPage/Overview/OverviewTopics.js]: ok

**Pull request merged**